stable/acs-engine-autoscaler/templates/deployment.yaml

```diff
@@ -63,6 +63,14 @@ spec:
             - --spare-agents
             - {{ .Values.acsenginecluster.spareagents }}
             {{- end }}
+            {{- if .Values.acsenginecluster.idlethreshold }}
+            - --idle-threshold
+            - {{ .Values.acsenginecluster.idlethreshold}}
+            {{- end }}
+            {{- if .Values.acsenginecluster.overprovision }}
+            - --over-provision
+            - {{ .Values.acsenginecluster.overprovision}}
```

Comment:
@wbuchwalter Can you pls take a look at how this value is passed in? I think it's looking for an integer and it's passing in a string?